examples/http/src/main/java/io/opentelemetry/example/http/HttpServer.java

```diff
@@ -40,40 +40,41 @@
 
 public class HttpServer {
 
-  private class HelloHandler implements HttpHandler {
+  private static class HelloHandler implements HttpHandler {
 
     @Override
-    public void handle(HttpExchange he) throws IOException {
+    public void handle(HttpExchange exchange) throws IOException {
       // Name convention for the Span is not yet defined.
       // See: https://github.com/open-telemetry/opentelemetry-specification/issues/270
       Span.Builder spanBuilder = tracer.spanBuilder("/").setSpanKind(Span.Kind.SERVER);
-      Span span = null;
 
       // Extract the context from the HTTP request
       Context ctx =
-          OpenTelemetry.getPropagators().getHttpTextFormat().extract(Context.current(), he, getter);
+          OpenTelemetry.getPropagators().getHttpTextFormat().extract(Context.current(), exchange, getter);
+
+      Span span = spanBuilder.startSpan();
       try (Scope scope = ContextUtils.withScopedContext(ctx)) {
```

Comment:
This should be with the new Span.

Reply:
you are correct. this example had some issues.  :)

Reply:
Probably that was me. I updated this example in a hurry among all the Context Prop changes and may have missed important bits here and there ;(